source/shared/routes.js

```diff
@@ -0,0 +1,8 @@
+import { Route } from 'react-router';
+import createApp from './components/App';
+
+export default React => () => {
+  const App = createApp(React);
+
+  return <Route path="/" component={App} />;
```

Comment:
I don't have time to take a deep look at this right now, but there's a [full example of redux-simple-router usage](https://github.com/freeqaz/redux-simple-router-example) you can use for reference. Have you looked at that?